lib/config/app_config.dart

```diff
@@ -69,6 +69,7 @@ abstract class AppConfig {
   static const double borderRadius = 20.0;
   static const double columnWidth = 360.0;
   static const int fetchContactsLimit = 20;
+  static const String androidDownloadPath = '/storage/emulated/0/Download';
```

Comment:
I think it's only support for emulated Android. 

We need to implement for iOS, Android, Web

Reply:
According to sources I found in internet (https://www.androidphonesoft.com/blog/how-to-access-storage-emulated-0-android/#What_is_emulated_storage_in_Android) it says storage emulated is available on every android not just virtual devices. I did Android and Web implementation we still need to do iOS but cannot with my setup

Reply:
hmm, please use the dynamic way to get the path of download folder. Maybe you also have no permision to access this path in new Android API

Reply:
@imGok Any update?